localtypings/pxtarget.d.ts

```diff
@@ -49,6 +49,7 @@ declare namespace pxt {
         ignoreDocsErrors?: boolean;
         variants?: Map<AppTarget>; // patches on top of the current AppTarget for different chip variants
         queryVariants?: Map<AppTarget>; // patches on top of the current AppTarget using query url regex
+        unsupportedBrowsers?: string[] // list of unsupported browsers for a specific target (eg IE11 in arcade). check browserutils.js browser() function for strings
```

Comment:
i think we should specify a combo of browser + version (for IE, version == "*"), but we might have to drop older versions of other editors. Maybe change the format to something like ``unsupportedBrowsers?: { id: string;  }`` , this would allow to add a ``minVersion?: string`` field in the future.

Reply:
or use a map, 
```{
   ie11: "*"
}```

Reply:
yeah, that makes sense. i think we could do minVersion as a number and if no minVersion is specified, it's treated as "*"